geojson/somalia_regions.geojson

```diff
@@ -0,0 +1,25 @@
2425 ], [ 42.85833, 10.23407 ], [ 42.85336, 10.24254 ], [ 42.83263, 10.27788 ], [ 42.817451895000033, 10.318265772000075 ], [ 42.80483, 10.35185 ], [ 42.80325, 10.35821 ], [ 42.79185, 10.4041 ], [ 42.76308, 10.45671 ], [ 42.75691, 10.47156 ], [ 42.75612, 10.47469 ], [ 42.74667, 10.51225 ], [ 42.73944, 10.52115 ], [ 42.73165, 10.52639 ], [ 42.71687, 10.53039 ], [ 42.70405, 10.53896 ], [ 42.6984, 10.54621 ], [ 42.69542, 10.55823 ], [ 42.69039, 10.5723 ], [ 42.68317, 10.59253 ], [ 42.68301, 10.60633 ], [ 42.68502, 10.61296 ], [ 42.69296, 10.6252 ], [ 42.7032, 10.63601 ], [ 42.70352986000006, 10.636268447000077 ], [ 42.7226, 10.65121 ], [ 42.722786821000057, 10.651510866000024 ], [ 42.73867, 10.67709 ], [ 42.76653, 10.72197 ], [ 42.78857, 10.75068 ], [ 42.7975, 10.76451 ], [ 42.81088, 10.78031 ], [ 42.81127, 10.78085 ], [ 42.84093, 10.82286 ], [ 42.85571, 10.83816 ], [ 42.86422, 10.85429 ], [ 42.87044, 10.86128 ], [ 42.87626, 10.87149 ], [ 42.88554, 10.8831 ], [ 42.91005, 10.91373 ], [ 42.93133, 10.94796 ], [ 42.961504853000065, 10.984247476000064 ], [ 42.968055, 10.997431 ], [ 42.997932972000058, 11.044779942000048 ], [ 43.261199851000072, 11.461990598000057 ], [ 43.265970624000033, 11.468165793000026 ], [ 43.273521423000034, 11.468915939000055 ], [ 43.287467957000047, 11.47083282400007 ], [ 43.293102265000073, 11.469118119000029 ], [ 43.297428131000061, 11.463465691000067 ], [ 43.301670074000072, 11.461489677000031 ], [ 43.305850983000028, 11.462271690000023 ], [ 43.306079915000055, 11.462952082000072 ], [ 43.306941986000027, 11.465514183000039 ], [ 43.306152344000054, 11.469405175000077 ], [ 43.300064087000067, 11.480767250000042 ], [ 43.297210693000068, 11.483231545000024 ], [ 43.284534454000038, 11.486862183000028 ], [ 43.286334991000047, 11.489661217000048 ], [ 43.286807774000067, 11.489749648000043 ], [ 43.290515900000059, 11.49044322900005 ], [ 43.302295685000047, 11.485185623000064 ], [ 43.307559967000032, 11.47932243300005 ], [ 43.313659668000071, 11.467761993000067 ], [ 43.315113068000073, 11.46500396700003 ] ] ], [ [ [ 43.338615417000028, 11.447710991000065 ], [ 43.334999085000049, 11.447633744000029 ], [ 43.356281281000065, 11.466810227000053 ], [ 43.359699249000073, 11.475798607000058 ], [ 43.363792420000038, 11.495502472000055 ], [ 43.364467428000069, 11.495024955000076 ], [ 43.367465973000037, 11.492903709000075 ], [ 43.362548828000058, 11.469617844000027 ], [ 43.360018185000058, 11.464345360000038 ], [ 43.359954834000064, 11.464213372000074 ], [ 43.338615417000028, 11.447710991000065 ] ] ], [ [ [ 43.455993652000075, 11.494044304000056 ], [ 43.451393127000074, 11.49118614200006 ], [ 43.450389862000065, 11.494383812000024 ], [ 43.452121735000048, 11.500398636000057 ], [ 43.459167481000065, 11.508595466000031 ], [ 43.459616493000055, 11.508829185000025 ], [ 43.464698791000046, 11.511474609000061 ], [ 43.46638488800005, 11.508981705000053 ], [ 43.456267632000049, 11.494438150000065 ], [ 43.455993652000075, 11.494044304000056 ] ] ] ] } },
```

Comment:
Do we need 'rotation'?

Reply:
Added to cater to some slanting region names that might need rotation. null for now because  frequency labels did not need it.